lib/commands/backup.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+const Command = require('../command');
+const Promise = require('bluebird');
+const createDebug = require('debug');
+const Zip = require('adm-zip');
+const fs = require('fs');
+const path = require('path');
+const cli = require('../');
+
+const debug = createDebug('ghost-cli:backup');
+
+class BackupCommand extends Command {
+    run(argv) {
+        const instance = this.system.getInstance();
+        const config = instance.config.values;
+        const datetime = (new Date()).toJSON().substring(0, 10);
+
+        let exporter;
+        let saveLocation = process.cwd();
+
+        // @todo check if we need to ensure we're in a ghost directory
+
+        if (argv.output) { // @todo add o as an alias
```

Comment:
this todo can be removed as well - you've added `o` as an alias down below, and yargs handles populating the args object correctly using aliases, so if you specify `--output <asdf>` or `-o <asdf>`, the args object will have both `o` and `output` populated with the value in both cases.